dbg win openblas

tylerjereddy opened the pull request:
debug

AtsushiSakai committed:
DOC: add Examples for stats.mstats.argstoarray() docstring.

AtsushiSakai committed:
fix doctest

rgommers committed:
Merge pull request #11983 from AtsushiSakai/issu_7168_3

DOC: add Examples for stats.mstats.argstoarray() docstring.

pv commented on azure-pipelines.yml:
```diff
@@ -101,26 +50,42 @@ jobs:
+      echo "library_dirs = d:\a\1\s\openblaslib" >> site.cfg
+      echo "include_dirs = d:\a\1\s\openblaslib" >> site.cfg
+      echo "runtime_library_dirs = d:\a\1\s\openblaslib" >> site.cfg
+      cat site.cfg
```
I think you also need to add the library paths for  `[openblas]` in addition to `[openblas64_]`, because the build needs both 32-bit and 64-bit libraries.

tylerjereddy replied:
I pushed that change in

tylerjereddy committed:
TST, CI: Azure Windows path fixups

* OpenBLAS install is now on the
same Windows root path as the SciPy clone
in Azure CI to avoid complications related to
NumPy gh-12530
